push.py

```diff
@@ -10,29 +10,33 @@
 """
 
 import argparse, os, sys, time
-from common import ccall
+from common import ccall, capture_output
 
 # Parsing flags
 parser = argparse.ArgumentParser(description='Build local.')
 parser.add_argument('-b', '--branch', help="log choosen preCICE branch")
 parser.add_argument('-t', '--test', help="choose system tests you want to use")
 parser.add_argument('-s', '--success', action='store_true' ,help="only upload log file")
+parser.add_argument('-o', '--os', type=str,help="Version of Ubuntu to use", choices =
```

Comment:
Please implement that in a generic way, not only for Ubuntu, there are other Linux versions in existence.